### ISSUE TITLE
Training vs validation loss

### ISSUE THREAD

**Issue by mohantyk** [mohantyk/deep-learning-book#1]
The training loss is more than the validation loss. See plot below:

https://github.com/mohantyk/deep-learning-book/blob/1e7d45329224d0e6e8010195425bcadd4878453d/vgg-data-augmentation.ipynb#L356-L365

**Comment by mohantyk** [comment]
Unable to reproduce in next run with gpu. Probably caused by the data augmentation for the training images. Latest commit of *vgg-data-augmentation* does not have the issue.
